aerogear/data-sync-ui
# Aerogear 7869 embed playground

## Motivation

This is the wiring for loading the playground react component.  

## What

This has the playground react component and it is hardwired to the localhosts graphql.  You can change this value by setting the endpoint in QueriesContainer.js


## Verification Steps
Make sure you are dunning the memeolist data-sync server on localhost:8080.
run this project (npm run server)
navigate to localhost:8080
navigate to Query
play!
## Checklist:


## Progress

- [x] Mount tab inside UI
- [x] Test tab with server side by copying URL

## Additional Notes

![screenshot from 2018-09-07 11-29-12](https://user-images.githubusercontent.com/2413816/45228522-61772380-b291-11e8-9102-60b133921ddd.png)
